src/new-client/src/App/Trades/TradesHeader/QuickFilter.tsx

```diff
@@ -33,6 +33,11 @@ const QuickFilterInput = styled("input")`
     color: ${({ theme }) => theme.core.textColor};
     opacity: 1;
   }
+
+  //Removes default clear icon
+  ::-webkit-search-cancel-button {
```

Comment:
non-webkit? have you tried on Firefox?

Reply:
This is a webkit specific issue, doesn't appear on Firefox :smile: